docs/software/python-cli/index.mdx

```diff
@@ -541,7 +541,7 @@ meshtastic --ble-scan
 
 ### --noproto
 
-Don't start the API, just function as a dumb serial terminal. Probably not very helpful from the command line. Used more for testing/internal needs.
+Don't start the API, just function as a dumb serial terminal. Useful for debugging because it doesn't count as a client. Depends on a physically cabled serial connection. It will connect but not display information over a network (--host) or Bluetooth (--ble) connection.
```

Comment:
I'll copy this version back into the `--help` output proper when I get a chance. This whole section is pretty much a copy-paste from there, so worth keeping them in sync. I've been meaning to do a refresh of this page since merging https://github.com/meshtastic/python/pull/679 anyway (which organizes the gigantic list of arguments a bit, haha).